website/static/files/build_orca_inventory_refined.sql

```diff
@@ -0,0 +1,140 @@
+    , version         text NOT NULL
+    , CONSTRAINT PK_collections PRIMARY KEY(collection_id)
+    , CONSTRAINT UNIQUE_collections UNIQUE (shortname, version)
+    );
+
+    COMMENT ON TABLE collections
+      IS 'Collections that are in the ORCA archive holdings.';
+    COMMENT ON COLUMN collections.collection_id
+      IS 'Collection ID from Cumulus usually in the format shortname__version.';
+    COMMENT ON COLUMN collections.shortname
+      IS 'Collection short name from Cumulus';
+    COMMENT ON COLUMN collections.version
+      IS 'Collection version from Cumulus';
+
+    -- Create the Providers Collection XREF table
+    CREATE TABLE IF NOT EXISTS provider_collection_xref
+    (
+      provider_id     text NOT NULL
+    , collection_id   text NOT NULL
+    , CONSTRAINT PK_provider_collection_xref PRIMARY KEY(provider_id,collection_id)
+    , CONSTRAINT provider_collection_fk FOREIGN KEY(provider_id) REFERENCES providers(provider_id)
+    , CONSTRAINT collection_provider_fk FOREIGN KEY(collection_id) REFERENCES collections(collection_id)
+    );
+
+    COMMENT ON TABLE provider_collection_xref
+      IS 'Cross refrence table that ties a collection and provider together and resolves the many to many relationship.';
+    COMMENT ON COLUMN provider_collection_xref.provider_id
+      IS 'Provider ID from the providers table.';
+    COMMENT ON COLUMN provider_collection_xref.collection_id
+      IS 'Collection ID from the collections table.';
+
+
+    -- Create the Granules table
+    CREATE TABLE IF NOT EXISTS granules
+    (
+      id                  int8 NOT NULL
+    , collection_id       text NOT NULL
+    , cumulus_granule_id  text NOT NULL
+    , execution_id  	  text NOT NULL
+    , ingest_time         timestamp with time zone NOT NULL
+    , last_update         timestamp with time zone NOT NULL
+    , CONSTRAINT PK_granules PRIMARY KEY(id)
+    , CONSTRAINT UNIQUE_granules UNIQUE (collection_id, cumulus_granule_id)
+    , CONSTRAINT collection_granule_fk FOREIGN KEY(collection_id) REFERENCES collections(collection_id)
+    );
+
+    COMMENT ON TABLE granules
+      IS 'Granules that are in the ORCA archive holdings.';
+    COMMENT ON COLUMN granules.id
+      IS 'Internal orca granule ID pseudo key';
+    COMMENT ON COLUMN granules.collection_id
+      IS 'Collection ID from Cumulus that refrences the Collections table.';
+    COMMENT ON COLUMN granules.cumulus_granule_id
+      IS 'Granule ID from Cumulus';
+    COMMENT ON COLUMN granules.execution_id
+      IS 'Step function execution ID from AWS';
+    COMMENT ON COLUMN granules.ingest_time
+      IS 'Date and time the granule was originally ingested into ORCA.';
+    COMMENT ON COLUMN granules.last_update
+      IS 'Last time the data for the granule was updated. This generally will coincide with a duplicate or a change to the underlying data file.';
+
+    -- Create the Files table
+    CREATE TABLE IF NOT EXISTS files
+    (
+      id                          int8 NOT NULL
```

Comment:
`int8` for `id` here should be `bigserial` to match what is in the db_deploy orca_sql.py

Reply:
Side branch